README.md

```diff
@@ -0,0 +1,5 @@
+# Humanizing Code Review
```

Comment:
Thanks for adding this README. It will be super helpful to the team moving forward ❤️.

<img src=https://media.giphy.com/media/mbhseRYedlG5W/giphy.gif />